Tests/Integration/DevAddrCacheTest.cs

```diff
@@ -902,5 +903,6 @@ public async Task Full_Update_Perform_Correctly_On_Non_Empty_Cache_And_Keep_Old_
         }
 
         private static DevAddr CreateDevAddr() => new DevAddr((uint)RandomNumberGenerator.GetInt32(int.MaxValue));
+        private static DevEui CreateDevEui() => TestEui.GenerateDevEui();
```

Comment:
nit: why not inline this?

Reply:
Done with 37e6dc642c7aac384bd0db4b8204c0aaf4c69cb9